### ISSUE TITLE
Stateless apps definition

### ISSUE THREAD

**Issue by CathPag** [cncf/glossary#250]
We need help with the definition of the stateful app. [Here is what we currently have](https://github.com/cncf/glossary/blob/main/content/en/stateful_apps.md) which is too technical for this glossary (and doesn't follow the current [style guide](https://glossary.cncf.io/style-guide/)). Below is our attempt. Although a lot simpler, it may not be 100% accurate. Please make sure to read the style guide before getting started.

**Comment by JatinNarang1** [comment]
@CathPag I would like to work on improving the glossary according to the style guide, if you approve me to do so😃

**Comment by CathPag** [comment]
Of course, @JatinNarang1. Please try to mirror [stateful apps ](https://glossary.cncf.io/stateful_apps/) a little bit since people will likely read them both. Thank you!

**Comment by JatinNarang1** [comment]
@CathPag Everything else apart from the definition seemed right, So I proposed minimal changes in the definition. Please do let me know if any further changes are required.. And would also love to contribute at any other kubernetes related projects :)

**Comment by CathPag** [comment]
Sorry, the link in the first comment was wrong. We need a definition for stateless, not stateful apps. [This is the definition that needs to be improved](https://glossary.cncf.io/stateless_apps/).

**Comment by 3V3RYONE** [comment]
Hey @CathPag, If you feel the glossary for stateless app needs to be updated direly, you may assign me to it. I would like to work on it :D

**Comment by CathPag** [comment]
@3V3RYONE, so @JatinNarang1 submitted a PR but it's not really mirroring the stateful app definition yet ([please see comment on PR](https://github.com/cncf/glossary/pull/259)). Let's see if Jatin wants to give it another try or hand it over to. you. 

Apologies for the late reply to both of you — it's been a crazy week!

**Comment by CathPag** [comment]
@3V3RYONE, I haven't heard from @JatinNarang1. Are you still up for working on this?

**Comment by 3V3RYONE** [comment]
@CathPag , Sure.. I would like to work on it.. you may assign me :D  
My exams would end in next 3 days, and after that I will give a PR! :))

**Comment by CathPag** [comment]
Excellent, it's all yours, @3V3RYONE! Good luck with your exams!

**Comment by 3V3RYONE** [comment]
Thank you, Cath!.. I am gonna do the PR tomorrow :))

**Comment by 3V3RYONE** [comment]
Hey @CathPag , I have done the PR for this issue.  
Sorry for the delay :( and let me know for any changes! :)

**Comment by JasonMorgan** [comment]
Hey folks, if anyone is interested this issue is up for grabs again.

**Comment by cnu1812** [comment]
Hi @JasonMorgan I would like to work on this. Please assign for me!

**Comment by CathPag** [comment]
There you go, @cnu1812. Please make sure to read the [style guide](https://glossary.cncf.io/style-guide/) before getting started.

**Comment by JasonMorgan** [comment]
@cnu1812 can you share a link to the PR once you have one?

**Comment by cnu1812** [comment]
Definitely! I'll share it by tomorrow.

**Comment by CathPag** [comment]
@JasonMorgan will try to bring this one over the finish line.

**Comment by CathPag** [comment]
Thanks, @PrathyushaModala. This is a tough one, and we'll need an experienced technologist to get it right. We've seen strong interest from students in contributing. If you are a student as well, I would leave this to someone with real-world experience. If you are an engineer with deep knowledge of the space, I'm happy to assign you this one.

Just trying to avoid a frustrating contributor experience :)

**Comment by FavourDaniel** [comment]
> We need help with the definition of the stateful app. [Here is what we currently have](https://github.com/cncf/glossary/blob/main/content/en/stateful_apps.md) which is too technical for this glossary (and doesn't follow the current [style guide](https://glossary.cncf.io/style-guide/)). Below is our attempt. Although a lot simpler, it may not be 100% accurate. Please make sure to read the style guide before getting started.

Is this still open? I would love to contribute

**Comment by CathPag** [comment]
Sure! Please make sure to read the [style guide](https://glossary.cncf.io/style-guide/) before getting started.

**Comment by FavourDaniel** [comment]
> Sure! Please make sure to read the [style guide](https://glossary.cncf.io/style-guide/) before getting started.

Thank you, on it.

**Comment by sergeyborovoy** [comment]
Hi @FavourDaniel  @CathPag  do we have any news on this one?

**Comment by CathPag** [comment]
Haven't heard from @FavourDaniel since early October, meaning this is ready to be released. Wanna work on it @sergeyborovoy?

**Comment by mikekwabs** [comment]
@CathPag  Hi, I would love to work on this issue.

**Comment by sergeyborovoy** [comment]
@CathPag I see @mikekwabs is enthusiastic about this one, don't mind him getting it :)

**Comment by CathPag** [comment]
Here you go, @mikekwabs. Please make sure to read all documentation before getting started.

**Comment by mikekwabs** [comment]
@CathPag Thank you.

**Comment by bhanuteja1901** [comment]
Hey!! i like to contribute 
can you help me in this

**Comment by mikekwabs** [comment]
@CathPag was it okay?

**Comment by CathPag** [comment]
What is ok? Are you working on this, @mikekwabs?

**Comment by mikekwabs** [comment]
yeah, sent a PR last month

**Comment by CathPag** [comment]
I can't find it. Can you link it to this issue?

**Comment by mikekwabs** [comment]
@CathPag https://github.com/cncf/glossary/pull/1573/commits/41b7e6076f9ace73172e88f6af75a197fbc4751a

**Comment by skushagra9** [comment]
Hey @CathPag, I would love to contribute

**Comment by CathPag** [comment]
I already assigned you to an issue, @skushagra9. Please work on that one first.

**Comment by FavourDaniel** [comment]
@CathPag I made a pull request to this #1678, can you please review?

**Comment by FavourDaniel** [comment]
@CathPag @iamNoah1 Please refer to this now #1731 instead of the previous one

**Comment by iamNoah1** [comment]
Gonna close this one as we have a PR merged already regarding this topic
